README.md

```diff
@@ -1,11 +1,11 @@
 ﻿# Processing, Compilation, and Execution of Expression Trees at Runtime
 
 C# provides the ability to write lambda expressions such as `n => n / 2`. The
-right hand of such an expression can be processed at runtime. This projects provides tools for such processing.
+right hand of such an expression can be processed at runtime. These projects provides tools for such processing.

```

Comment:
It's really just the `ExpressionUtils` project actually. Nonetheless, "These" does not work with "provides"